packages/babel-cli/src/babel/dir.js

```diff
@@ -43,7 +44,9 @@ export default async function({
         ? path.extname(relative)
         : cliOptions.outFileExtension,
     );
-
+    if (!print) {
+      babelOptions.showConfig = false;
```

Comment:
maybe it's just a preference but I would use ```
babelOptions.showConfig = print;
```

and add a default value to `print`